Automattic/wp-calypso
# Gutenboarding: Update template for Reynolds FSE design

The original Reynolds design shared the same template name as the Reynolds FSE design. This led to references to the same starter site and identical thumbnail previews, even though the FSE and non-FSE design had distinct differences.

Requires D48345-code

#### Changes proposed in this Pull Request

* The Reynolds (FSE) design was previously referencing a starter site that was using the `Rockfield` theme. 
* The Reynolds (FSE) design should now point to `https://reynoldsfullsiteeditingstarter.wordpress.com`, which is a site editor enabled site implementing the `seedlet-blocks` theme.

#### Known Issues (aka Help Request 🙂 )
* Changing fonts in the live preview doesn't work. I haven't investigated it yet, * but maybe there's a very simple reason.

#### Testing instructions



* Apply D48345-code and sandbox the API.
* Run Calypso and open `http://calypso.localhost:3000/new?flags=gutenboarding/site-editor`.
* Ensure that creating a site editor site with the Reynolds (FSE) design still works as expected

Fixes #44669
